<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1615979782133' stop='1615979865098'>
    <name>Check sign in</name>
    <title>Check sign in</title>
    <test-cases>
        <test-case start='1615979782135' status='failed' stop='1615979812906'>
            <name>should compare two phones</name>
            <title>should compare two phones</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (internal/timers.js:554:17)
                    at processTimers (internal/timers.js:497:7)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1615979812907' status='failed' stop='1615979824156'>
            <name>should delete items from the comparison list</name>
            <title>should delete items from the comparison list</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected 'https://yandex.by/' to equal 'https://market.yandex.by/?clid=505&amp;utm_source=main_stripe_big'.</message>
                <stack-trace>Error: Failed expectation
                    at C:\Users\User\Protractor_project\secondPull\test\test.spec.js:41:45
                    at ManagedPromise.invokeCallback_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1615979824156' status='failed' stop='1615979842044'>
            <name>should sort cameras by price</name>
            <title>should sort cameras by price</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected 'https://yandex.by/' to equal 'https://market.yandex.by/?clid=505&amp;utm_source=main_stripe_big'.</message>
                <stack-trace>Error: Failed expectation
                    at C:\Users\User\Protractor_project\secondPull\test\test.spec.js:66:45
                    at ManagedPromise.invokeCallback_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1615979842044' status='failed' stop='1615979853973'>
            <name>should sort by tag</name>
            <title>should sort by tag</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected 'https://yandex.by/' to equal 'https://market.yandex.by/?clid=505&amp;utm_source=main_stripe_big'.</message>
                <stack-trace>Error: Failed expectation
                    at C:\Users\User\Protractor_project\secondPull\test\test.spec.js:89:45
                    at ManagedPromise.invokeCallback_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1615979853974' status='failed' stop='1615979865095'>
            <name>should check Yandex music</name>
            <title>should check Yandex music</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: No element found using locator: By(css selector, #passp-field-login)</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(css selector, #passp-field-login)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27
                    at ManagedPromise.invokeCallback_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as isEnabled] (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as isEnabled] (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at UserContext.&lt;anonymous> (C:\Users\User\Protractor_project\secondPull\test\test.spec.js:112:40)
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)
                From: Task: Run it("should check Yandex music") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at PatchedSpec.queueRunnerFactory (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at PatchedSpec.Spec.execute (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (C:\Users\User\Protractor_project\secondPull\test\test.spec.js:101:5)
                    at addSpecsToSuite (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (C:\Users\User\Protractor_project\secondPull\test\test.spec.js:8:1)
                    at Module._compile (internal/modules/cjs/loader.js:1063:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
                    at Module.load (internal/modules/cjs/loader.js:928:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:769:14)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>